src/strucclustutils/structureconvertalis.cpp

```diff
@@ -522,6 +525,14 @@ int structureconvertalis(int argc, const char **argv, const Command &command) {
                     tmres = tmaligner->computeTMscore(targetCaData, &targetCaData[res.dbLen], &targetCaData[res.dbLen+res.dbLen], res.dbLen,
                                                       res.qStartPos, res.dbStartPos, Matcher::uncompressAlignment(res.backtrace));
                 }
+                LDDTcalculator::LDDTscoreResult lddtres;
+                if(needLDDT) {
+                    lddtcalculator = new LDDTcalculator(res.qLen, res.dbLen);
+                    lddtcalculator->initVariables(res.qLen, res.dbLen, res.qStartPos, res.dbStartPos, Matcher::uncompressAlignment(res.backtrace));
```

Comment:
I think we should have one method for initQuery that only gets (qLen, qx, qy, qz) as a parameter. This should initialize the query matrix.